packages/react-components/react-positioning/src/createPositionManager.ts

```diff
@@ -37,6 +37,7 @@ interface PositionManagerOptions {
  */
 export function createPositionManager(options: PositionManagerOptions): PositionManager {
   const { container, target, arrow, strategy, middleware, placement } = options;
+  let destroyed = false;
```

Comment:
```suggestion
  let isDestroyed = false;
```

nit: We use `is` prefix there already (isFirstUpdate)